articles/migrations/0086_articleprojectlink.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0003_remove_projectpage_name'),
+        ('articles', '0085_articlepage_json_file'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ArticleProjectLink',
```

Comment:
This migration exists, but I the the model it creates may already exist. I would recommend remove all of the migration which have been created and then regenerating from scratch.